src/test/java/org/broadinstitute/consent/http/authentication/OAuthAuthenticatorTest.java

```diff
@@ -44,7 +43,7 @@ public class OAuthAuthenticatorTest {
 
     private OAuthAuthenticator oAuthAuthenticator;
 
-    @Before
+    @BeforeEach
     public void setUp() {
         openMocks(this);
```

Comment:
While you're at it, you may want to replace `openMocks(this)` with `@ExtendWith(MockitoExtension.class)`. Although this uses strict stubbing by default (which is good), so it may break your tests until you fix them (which is bad).

Reply:
Thanks - I'll take a look at that. It's definitely beyond the scope of what we're trying to do, but I can probably do that in a related PR.